global.d.ts

```diff
@@ -27,6 +27,8 @@ declare global {
   type ScrollView = DragonData & {
     in_garden: boolean;
     in_seed_tray: boolean;
+    incubated?: boolean;
+    stunned?: boolean;
```

Comment:
I'm guessing these being optional is because only either one or the other can be present, yes?

Reply:
Exactly